src/util/schemas/UserDeleteSchema.ts

```diff
@@ -0,0 +1,3 @@
+export interface UserDeleteSchema {
+	user_id: string;
+}
```

Comment:
Schema?? User id is available in the url...

Reply:
removed from the REST schema, will be kept for ability to ask deletion via gateway